src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -15,11 +15,6 @@
 
 import java.util.List;
 
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.SwaggerDefinition;
-import io.swagger.annotations.Tag;
-import org.hibernate.validator.constraints.SafeHtml;
 import org.slf4j.Logger;
```

Comment:
Isn't " import io.swagger.annotations.Api; " needed? since we use "@API" annotations for Swagger.